ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/pipes/noData.pipe.ts

```diff
@@ -7,7 +7,7 @@ import {Pipe, PipeTransform} from '@angular/core';
 export class NoDataPipe implements PipeTransform {
   transform(value: any): string {
     const tempElement = document.createElement('div');
-    tempElement.innerHTML = value || '&nbsp;';
+    tempElement.innerHTML = (typeof value === 'number' ? String(value) : value) || '&nbsp;';
     return tempElement.innerText;
```

Comment:
Seems like you can use just `String.fromCharCode(160)` for non breaking space 
(see [link](https://stackoverflow.com/questions/5237989/how-is-a-non-breaking-space-represented-in-a-javascript-string))

Reply:
Better, Thank you

Reply:
May be, just enough to have smth like:

```
transform(value: any): string {
    const nonBreakingSpace = String.fromCharCode(160);
    return (value || value === 0) ? String(value) : nonBreakingSpace;
}
```